src/renderer/i18n/en.ts

```
@@ -777,8 +777,8 @@
~  presentationStyle: {
     styleTemplate: {
       title: "Style template",
       description:
-        "Overwrite the style. Be careful as the existing style settings and character settings will be lost.",
+        "Overwrite the style. Be careful as the existing style settings and material settings will be lost.",
```

ESLint

[error] Delete `⏎·······`

(prettier/prettier)